source/extensions/transport_sockets/tls/context_impl.h

```diff
@@ -73,7 +73,7 @@ class ContextImpl : public virtual Envoy::Ssl::Context {
    * @param pattern the pattern to match against (*.example.com)
    * @return true if the san matches pattern
    */
-  static bool dNSNameMatch(const std::string& dnsName, const char* pattern);
```

Comment:
Huh, this name was already out of date.  Mind fixing it as long as we're in here?